exalearn-ui/src/app/services/roles.service.ts

```diff
@@ -0,0 +1,47 @@
+import {Injectable} from '@angular/core';
+import {RedirectBtn} from "../interfaces/interfaces";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class RolesService {
+  private userBtns: readonly RedirectBtn[] = [
+    {name: "my tests history", url: "/history"},
```

Comment:
We use single quotes everywhere, please, try to refactore code with prettier.
@HrebenYury, did you configure git pre-commit hook? Or we didn't have single-quote rule?

Reply:
changed

Reply:
> @HrebenYury, did you configure git pre-commit hook? Or we didn't have single-quote rule?

No, I didn't configure pre-commit hook. I will solve this problem as soon as possible. And we have single-quote rule into .prettierrc file.